### PR TITLE
Add margin example

### PR DESCRIPTION
**Pull request by wbamberg**
This is a pretty simple example, feedback welcome!

### CODE REVIEW

**Comment by wbamberg** [comment]
This also uses very similar colors and borders to the box-sizing example. I wonder if we should try to standardise these?

**Comment by schalkneethling** [comment]
> This also uses very similar colors and borders to the box-sizing example. I wonder if we should try to standardise these?

I am all for standards and less code to maintain :) Can probably create a CSS file called `examples-common.css`

File: live-examples/css-examples/margin.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="margin">
```
Comment (schalkneethling) [inline live-examples/css-examples/margin.html]: I am thinking that, if we know that the property is well supported given our browser support matrix, we do not have to add the data attribute.

File: live-examples/css-examples/margin.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code id="example_five" class="language-css">margin: 0;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_five">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
```
Comment (schalkneethling) [inline live-examples/css-examples/margin.html]: Nit: Empty line not required

File: live-examples/css-examples/margin.html (hunk 1 above)
Comment (wbamberg) [reply live-examples/css-examples/margin.html]: At the moment it seems that if this is omitted, the editor assumes that the property is not supported, and just displays the static example. So I have left this in.

Comment (schalkneethling) [reply live-examples/css-examples/margin.html]: Oh that is certainly a bug. I will look into it.

Comment (wbamberg) [reply live-examples/css-examples/margin.html]: It should be fixed in https://github.com/mdn/interactive-examples/pull/381.

**Pull request merged**